New version: ITensors v0.8.0

JuliaRegistrator opened the pull request:
- Registering package: ITensors
- Repository: https://github.com/ITensor/ITensors.jl
- Created by: @mtfishman
- Version: v0.8.0
- Commit: dbda166df4ce2edae5778cefd344d809cc1271d7
- Reviewed by: @mtfishman
- Reference: https://github.com/ITensor/ITensors.jl/commit/dbda166df4ce2edae5778cefd344d809cc1271d7#commitcomment-152101836
- Description: A Julia library for efficient tensor computations and tensor network calculations. ITensor is supported by the Simons Foundation's Flatiron Institute.
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

- This release should not be breaking to the average user using documented features of the library. This removes internal submodules that held experimental code for rewriting the internals of NDTensors.jl/ITensors.jl, which have now been turned into separate packages for future development. It is marked as breaking since ITensorMPS.jl was making use of some that experimental code, and will be updated accordingly.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd62eecefa0aba72487fceb23c82f19d8bfbc5e475fab7a3967e39de58989abaf73ef6d31f9a90d30ded42dc8885e93dba76f54728edf9e72cf131bc4112a4d8e2a5017c513863286c453bddc92ca0ecf5afac67fdf3c7e4e944dd1e33802069746200e6738326f0038c04658503ba74293fde9e0016f4d7f3e9b57b09fa5f604395bd4a2ba3df83e2cdbe881ff5480e4817e07de2ac382ede2ce10387a45a443009a06850e3281ed0f8299f5f95ac8495f -->

mtfishman commented:
This PR was a mistake, I plan to register a different commit after a new version of a dependent package is registered (https://github.com/JuliaRegistries/General/pull/124280).

JuliaRegistrator committed:
New version: ITensors v0.8.0

UUID: 9136182c-28ba-11e9-034c-db9fb085ebd5
Repo: https://github.com/ITensor/ITensors.jl.git
Tree: 583de9926e696f76553c8118aafc070beadb0c02

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
